Makefile

```diff
@@ -3,22 +3,21 @@ IMAGE_REPOSITORY ?= smartxworks
 CAPCH_ROOTFS_IMAGE ?= $(IMAGE_REPOSITORY)/capch-rootfs-$(KUBERNETES_VERSION)
 CAPCH_ROOTFS_CDI_IMAGE ?= $(IMAGE_REPOSITORY)/capch-rootfs-cdi-$(KUBERNETES_VERSION)
 
-all: build push
+all: build
 
 .PHONY: build
 build:
 	iidfile=$$(mktemp /tmp/iid-XXXXXX) && \
 	imagesfile=$$(mktemp /tmp/images-XXXXXX) && \
-	docker build --build-arg KUBERNETES_VERSION=$(KUBERNETES_VERSION) --target rootfs -f rootfs/Dockerfile --iidfile $$iidfile . && \
+	docker buildx build --build-arg KUBERNETES_VERSION=$(KUBERNETES_VERSION) --target rootfs -f rootfs/Dockerfile --iidfile $$iidfile --load . && \
 	docker run --rm $$(cat $$iidfile) kubeadm config images list --kubernetes-version $(KUBERNETES_VERSION) > $$imagesfile && \
-	cat $$imagesfile | xargs -L 1 ctr image pull && \
-	cat $$imagesfile | xargs ctr image export rootfs/images-$(KUBERNETES_VERSION).tar && \
+	cat $$imagesfile | xargs -L 1 ctr image pull --platform linux/amd64 && \
+	cat $$imagesfile | xargs -L 1 ctr image pull --platform linux/arm64 && \
+	mkdir -p rootfs/linux/amd64 && \
+	mkdir -p rootfs/linux/arm64 && \
+	cat $$imagesfile | xargs ctr image export --platform linux/amd64 rootfs/linux/amd64/images-$(KUBERNETES_VERSION).tar && \
+	cat $$imagesfile | xargs ctr image export --platform linux/arm64 rootfs/linux/arm64/images-$(KUBERNETES_VERSION).tar && \
 	rm -rf $$iidfile $$imagesfile && \
-	docker build --build-arg KUBERNETES_VERSION=$(KUBERNETES_VERSION) --target virtink-container-rootfs -t $(CAPCH_ROOTFS_IMAGE) -f rootfs/Dockerfile . && \
-	docker build --build-arg KUBERNETES_VERSION=$(KUBERNETES_VERSION) -t $(CAPCH_ROOTFS_CDI_IMAGE) -f rootfs/Dockerfile . && \
-	rm -rf rootfs/images-$(KUBERNETES_VERSION).tar
-
-.PHONY: push
-push:
-	docker push $(CAPCH_ROOTFS_IMAGE)
-	docker push $(CAPCH_ROOTFS_CDI_IMAGE)
+	docker buildx build --platform linux/amd64,linux/arm64 --build-arg KUBERNETES_VERSION=$(KUBERNETES_VERSION) -o type=image,name=$(CAPCH_ROOTFS_IMAGE),push=true,registry.insecure=true -f rootfs/Dockerfile --target virtink-container-rootfs . && \
```

Comment:
the `-o ...` part should be removed?

Reply:
this is buildx format, it is same as `-t xx:tag --push`, as we need `registry.insecure=true` for dev, so I write it this way.

Reply:
there is not separate `make push` now,  buildx doesn't have a push command. on the other hand because we build both linux/amd64 and linux/arm64 in one `docker buildx build` command, the result is a docker manifest which can't be export to local docker and then push by local docker. so we have to push it in `docker buildx build`

Reply:
and there is another problem, I didn't find a way to make `docker buildx build --push` to our internal harbor registry, as `docker buildx` don't have command like `docker login`

Reply:
Docker buildx will use credentials created by docker login. And so make build should be make push?

Reply:
I added kernel build, now they are `make kernel` and `make rootfs`

Reply:
Change to `make push-kernel` and `make push-rootfs`? I think it would make it clear that this task would push images to registry